torch/csrc/jit/codegen/cuda/ir_builder.h

```diff
@@ -15,20 +16,6 @@ class Kernel;
 
 class IrCloner;
 
-// Passkey for builder to register properties with statements, and to call
```

Comment:
Moved to a separate file `torch/csrc/jit/codegen/cuda/ir_builder_key.h` so it can be included in `torch/csrc/jit/codegen/cuda/ir_base_nodes.h`

Reply:
Oh, I did that too in #2197 😄